ios/native/RCTAppDelegate+Reanimated.mm

```diff
@@ -0,0 +1,28 @@
+#if REACT_NATIVE_MINOR_VERSION >= 72 && !defined(RCT_NEW_ARCH_ENABLED) && !defined(DONT_AUTOINSTALL_REANIMATED)
+
+#import <RNReanimated/RCTAppDelegate+Reanimated.h>
+#import <RNReanimated/REAInitializer.h>
+#import <objc/runtime.h>
+
+@implementation RCTAppDelegate (Reanimated)
+
+- (std::unique_ptr<facebook::react::JSExecutorFactory>)swizzled_jsExecutorFactoryForBridge:(RCTBridge *)bridge
+{
+  reanimated::REAInitializer(bridge);
+  return [self swizzled_jsExecutorFactoryForBridge:bridge]; // call the original method
+}
+
++ (void)load
```

Comment:
Is using load vs initialize a common practice here?

Reply:
Honestly that's just what ChatGPT came up with.